builtin/rebase.c

```diff
@@ -119,6 +124,10 @@ static int run_specific_rebase(struct rebase_options *opts)
 				oid_to_hex(&opts->onto->object.oid));
 	argv_array_pushf(&env, "onto_name=%s", opts->onto_name);
 	argv_array_pushf(&env, "revisions=%s", opts->revisions);
+	argv_array_pushf(&env, "GIT_QUIET=%d", opts->quiet);
+	argv_array_pushf(&env, "git_am_opt=%s", opts->git_am_opt);
+	argv_array_pushf(&env, "verbose=%d", opts->verbose);
```

Comment:
That would be a change in behavior. The `verbose` variable is expected to be empty by default (indicating `false`): https://github.com/prertik/git/blob/ae91dbb2556d7ea9f4e205fbf7847fa8579c64b4/TODO-rebase.sh#L75. It can also be `t` to indicate `true`: https://github.com/prertik/git/blob/ae91dbb2556d7ea9f4e205fbf7847fa8579c64b4/TODO-rebase.sh#L359

Please use this instead:

```c
argv_array_push(&env, opts->verbose ? "verbose=t" : "verbose=");
```

Same goes for `diffstat`.